frontend/src/app/components/filters/filter-boolean-value/filter-boolean-value.component.ts

```diff
@@ -33,7 +33,8 @@ import {QueryFilterInstanceResource} from 'core-app/modules/hal/resources/query-
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/a6d121d5804683a31696c23069b691a8cfc94e9d/frontend/src/app/components/filters/filter-boolean-value/filter-boolean-value.component.ts#L33"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>